libcontainer/system/kernelparam/lookup.go

```diff
@@ -0,0 +1,52 @@
+package kernelparam
+
+import (
+	"io/fs"
+	"strings"
+	"unicode"
+)
+
+type KernelBootParam string
+
+const (
+	IsolatedCPUAffinityTransition KernelBootParam = "runc.exec.isolated-cpu-affinity-transition"
+	NohzFull                      KernelBootParam = "nohz_full"
```

Comment:
I'd just use the `string` directly, and remove these consts (move them to code that's using `LookupKernelBootParameters`)

Reply:
fixed